extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/BackChannelLogoutHandler.java

```diff
@@ -160,7 +162,12 @@ private TenantConfigContext getTenantConfigContext(RoutingContext context) {
         private boolean isMatchingTenant(String requestPath, TenantConfigContext tenant) {
             return tenant.oidcConfig.isTenantEnabled()
                     && tenant.oidcConfig.getTenantId().get().equals(oidcTenantConfig.getTenantId().get())
-                    && requestPath.equals(tenant.oidcConfig.logout.backchannel.path.orElse(null));
+                    && requestPath.equals(getRootPath() + tenant.oidcConfig.logout.backchannel.path.orElse(null));
```

Comment:
I'm not sure if the ``.orElse(null)`` makes sense when concatenating a string...

Reply:
@AB-xdev Quarkus http root is never null, so it should be ok